limitador/src/storage/atomic_expiring_value.rs

```diff
@@ -141,4 +141,11 @@ mod tests {
         });
         assert!([2i64, 3i64].contains(&atomic_expiring_value.value.load(Ordering::SeqCst)));
     }
+
+    #[test]
+    fn size_of_struct() {
+        // This is ugly, but we don't have access to `AtomicExpiringValue` in the server,
```

Comment:
No. That's the actual size, i.e. `2 * size_of::<u64>() == 2 * 8 == 16`
This test is just here because the value is hardcoded in the server's `main.rs`, as I can't reference `AtomicExpiringValue` from there… So, if we ever happen to make this bigger (or smaller), this test will fail as a reminded to update the value in [here](https://github.com/Kuadrant/limitador/pull/195/files#diff-77c4c3afe1e75f3f1602389f5895694a70395b1a7450ad7776945631b4bd28e9R815)